smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -557,58 +550,6 @@ def pollResponse(response, data) {
 	}
 }
 
-def getActivityList() {
```

Comment:
It's not necessary to remove this method unless it's no longer needed. The method missing exceptions were related to executing an old version of the installed smart app